src/main/java/lotto/domain/LottoBalls.java

```diff
@@ -18,10 +21,14 @@ public LottoBalls(String numbersString) {
         this.lottoBalls = createLottoBallList(numbersSplitted);
     }
 
+    public List<LottoBall> getLottoBalls() {
+        return lottoBalls;
```

Comment:
Collections.unmodifiableList 에 대해 한번 찾아보시고, 

```suggestion
        return Collections.unmodifiableList(lottoBalls);
```
현재 프로그램에서 어떤 차이가 있을지 고민해보면 좋을 것 같아요~
다른곳도 일급컬렉션으로 랩핑한 곳이 있다면 이를 활용하면 좋을지 생각해보면 좋을 것 같습니다.

Reply:
List 에 변경이 생길 가능성을 줄여주는 거군요!
변경이 되면 안되는 리스트를 유지하기 위해 사용하면 좋을 것 같아요. 협업할 때 혹시나 발생할 수 있는 변경 문제를 방지할 수 있어 유용하게 쓸 수 있겠네요! 팁 감사합니다 ㅎㅎ